package.json

```diff
@@ -2,11 +2,14 @@
   "name": "ink-docs",
   "version": "0.0.0",
   "private": true,
+  "license": "SEE LICENSE IN LICENSE",
   "scripts": {
     "docusaurus": "docusaurus",
     "start": "docusaurus start --port 8080",
-    "build": "docusaurus build",
-    "build:release": "ci-ghact-docs",
+    "lint": "yarn eslint .",
+    "fmt": "yarn prettier --write \"docs/**/*.{js,jsx,ts,tsx,mdx}\" \"src/**/*.{js,jsx,ts,tsx,mdx,css}\" && yarn eslint --fix .",
+    "build": "yarn lint && docusaurus build",
+    "build:release": "yarn lint && ci-ghact-docs",
```

Comment:
Is `yarn lint` in this script redundant? Does `ci-ghat-docs` call `build` on lint 11?